ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -341,6 +341,29 @@ export default function (
         })
       }
     },
+    panToShapesExtent() {
+      var extent = Openlayers.extent.createEmpty()
```

Comment:
What's an extent?

Reply:
As I understand it, it's a bounding box that contains a collection of geos